README.md

```diff
@@ -17,20 +17,20 @@
 Cockatrice is an open-source multiplatform supported program for playing tabletop card games over a network. The program's server design prevents any kind of client modifications to gain an unfair advantage in a game. The client also has a built in single-player mode where you can brew without being connected to a server. This project is written in C++ and is using the Qt5 libraries.<br>
 
 
-# Downloads
+# Downloads [![Cockatrice Eternal Download Count](https://img.shields.io/github/downloads/cockatrice/cockatrice/total.svg)](http://somsubhra.github.io/github-release-stats/?username=cockatrice&repository=cockatrice)
```

Comment:
It works for me with chrome and ff:
https://github.com/Cockatrice/Cockatrice/tree/tooomm-dl_badges#downloads-

Correctly opens http://somsubhra.github.io/github-release-stats/?username=cockatrice&repository=cockatrice - I'll add https for sure.